Feature/login view

andresarangopro opened the pull request.

andresarangoa committed:
feature/login_view

- made compose view for login
- fixed compose view for get started

andresarangoseasoned committed:
Merge remote-tracking branch 'origin/develop' into feature/login_view

andresarangoseasoned committed:
view fixed

-

andresarangoseasoned committed:
feature/login_view

- navigation fron viewmodels implemented

andresarangoseasoned committed:
feature/login_view

- Fixed navigation

juancamilov06 commented on app/src/main/java/com/growth/streetwarrior/custom/component/MailAndPass.kt:
```diff
@@ -0,0 +1,80 @@
+
+
+data class ParamsMailAndPassword(
+    val title: Int,
```
Annotate this with @StringRes to identify it as an string resource

juancamilov06 commented on app/src/main/java/com/growth/streetwarrior/presentation/ui/login/getStarted/GetStartedViewModel.kt:
```diff
@@ -0,0 +1,22 @@
+import javax.inject.Inject
+
+@HiltViewModel
+class GetStartedViewModel@Inject constructor(
```
Add an space between the class name and the Inject anonotation

andresarangoseasoned committed:
feature/login_view

- fixed space t in GetstartedViewmodel
- myRouteNavigator renamed to StreetWarriorNavigator
- added @StringRes to identify string resources
